packages/trip-details/src/types.ts

```diff
@@ -0,0 +1,43 @@
+import moment from "moment";
+// Prettier does not recognize the import type syntax.
+// eslint-disable-next-line prettier/prettier
+import type { Itinerary } from "@opentripplanner/types";
+
+export type CaloriesDetailsProps = {
+  bikeSeconds: number;
+  calories: number;
+  walkSeconds: number;
+};
+
+export type DepartureDetailsProps = {
+  departureDate: moment.Moment;
+};
```

Comment:
See notes above about getting rid of moment as a dependency.

Reply:
Good suggestion, thanks, removed in https://github.com/opentripplanner/otp-ui/pull/281/commits/a9adf69db199e0306ae19215a8bd88625b28634f.